src/page/NewCheck/__tests__/ApiEndPointChecks/TCPCheck/4-alerting.payload.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { config } from '@grafana/runtime';
+import { screen } from '@testing-library/react';
+import { PRIVATE_PROBE } from 'test/fixtures/probes';
+import { probeToMetadataProbe } from 'test/utils';
+
+import { CheckType, FeatureName } from 'types';
+import { goToSection, renderNewForm, submitForm } from 'page/__testHelpers__/checkForm';
+
+import { fillMandatoryFields } from '../../../../__testHelpers__/apiEndPoint';
+
+const checkType = CheckType.TCP;
+
+describe(`TCPCheck - Section 4 (Alerting) payload`, () => {
+  it(`can add TLS certificate expiry alert when TLS is enabled`, async () => {
+    jest.replaceProperty(config, 'featureToggles', {
+      // @ts-expect-error
```

Comment:
Why?

Reply:
I've removed the error suppressions [here](https://github.com/grafana/synthetic-monitoring-app/pull/1153/commits/5947f487812588bb216b9847ba64aca203eeadec) and created a utility function to override feature flags in tests 👍